relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -360,6 +360,18 @@ pub(crate) fn extract_tags(span: &Span, config: &Config) -> BTreeMap<SpanTagKey,
                 );
             }
```

Comment:
Same here: If `span.data` never actually contains the key `http.decoded_response_body_length`, you can remove the old if block.